PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/selection/SelectionPaneController.java

```diff
@@ -55,16 +59,37 @@
   private SimpleObjectProperty<ISelectable> primarySelection;
   private SimpleObjectProperty<ISelectable> secondarySelection;
 
+  /**
+   * Method wrapper for an exception.
+   *
+   * <p>
+   * This method calls {@link ISelectionInfo#getNode()} on the given {@link ISelectionInfo}
+   * and catches its {@link IOException}. In the latter case it will return a simple
+   * {@link Text} with an error.
+   * </p>
+   *
+   * @param info the given {@link ISelectionInfo}
+   * @return a {@link Node} that can be displayed
+   */
+  private Node safeGetDescription(ISelectionInfo info) {
```

Comment:
The thing is, the FXMLLoader will throw an IOException which in java the programmer SHOULD always catch as these exceptions simply might occur at runtime. 
As the only reasonable case in which this exception is thrown, the cause is most likely that the `.fxml` file does not exist. Other scenario's would be actual exceptions / errors with IO. When that kind of stuff goes wrong it is not worth it for our application to do something sensible besides not directly crashing. (IMHO)